### DISCUSSION TITLE
zipkinServices variable for SkyWalking 9.4.0

### DISCUSSION THREAD

**Discussion by Hu-WF** [apache/skywalking#10865] [Q&A] [answered]
Is the _zipkinServices_ variable (in the application. yml of the webapp) mandatory for SkyWalking v9.4.0? Does it mean that a zipkin backend must be started before starting SkyWalking v9.4.0?
![Uploading 捕获.PNG…]()

**Comment by wu-sheng** [comment]
We can't see your screenshot, and don't know what do you mean.

**Comment by Hu-WF** [comment]
I deployed SkyWalking 9.14.0, but the error is as follows:
![zipkin](https://github.com/apache/skywalking/assets/30825060/6f988eeb-765b-4f56-a663-b3845b667703)
Does it mean that a Zipkin backend must be deployed first：
![捕获](https://github.com/apache/skywalking/assets/30825060/bc04e3c6-d3db-4d1d-ad73-0a546b43052d)

**Comment by Hu-WF** [comment, marked as answer]
The problem has been resolved. It was caused by the configuration of the old version (v9.3.0) overwriting the new version (v9.4.0). Thank you for your reply.
